src/tools/other/gen_vx_mask/gen_vx_mask.cc

```diff
@@ -1551,13 +1658,21 @@ static void write_netcdf(const DataPlane &dp) {
    vector<float> mask_data(grid.nx()*grid.ny());
 
    // Loop through each grid point
-   for(int x=0; x<grid.nx(); x++) {
-      for(int y=0; y<grid.ny(); y++) {
-         n = DefaultTO.two_to_one(grid.nx(), grid.ny(), x, y);
-         mask_data[n] = (float) dp(x, y);
-      } // end for y
-   } // end for x
+#pragma omp parallel default(none) \
+   shared(grid, DefaultTO, mask_data, dp)
+   {
+
+#pragma omp for schedule(static) \
+                collapse(2)
 
+      for(int x=0; x<grid.nx(); x++) {
+         for(int y=0; y<grid.ny(); y++) {
+            int n = DefaultTO.two_to_one(grid.nx(), grid.ny(), x, y);
+            mask_data[n] = (float) dp(x, y);
+         } // end for y
+      } // end for x
+   } // end of parallel
+      
    if(!put_nc_data_with_dims(&mask_var, mask_data.data(), grid.ny(), grid.nx())) {
```

Comment:
@jprestop I'll note that I'm planning to replace this section with a call to a new `put_nc_data_plane_float()` utility function that I'm adding for #3145.

Reply:
Thanks!